eng/common/scripts/stress-testing/generate-scenario-matrix.ps1

```diff
@@ -17,24 +17,23 @@ function GenerateScenarioMatrix(
 ) {
     $yamlConfig = Get-Content $matrixFilePath -Raw
 
-    $prettySerializedMatrix = &"$PSScriptRoot/../job-matrix/Create-JobMatrix.ps1" `
+    $prettyMatrix = &"$PSScriptRoot/../job-matrix/Create-JobMatrix.ps1" `
         -ConfigPath $matrixFilePath `
         -Selection $Selection `
         -DisplayNameFilter $DisplayNameFilter `
         -Filters $Filters `
         -Replace $Replace `
         -NonSparseParameters $NonSparseParameters
-
-    $prettyMatrix = $prettySerializedMatrix | ConvertFrom-Json -AsHashtable
```

Comment:
Aha. I totally missed this line when trying to find an un-ordered deserialization.